src/store/metadata/stub/content/editions/persistent-absences-for-pupils_2023-05.json

```diff
@@ -0,0 +1,107 @@
+        "identifier": "2023-05",
+        "title": "Persistent absences for pupils looked after by local authorities",
+        "summary": "Percentage of pupils who have been looked after by the local authorities over financial years 2016/17 - 2021/22.",
+        "description": "This dataset shows the percentage of pupils in state-funded primary, state funded secondary, special schools and pupil referral units who have been looked after continuously for at least 12 months (as of 31 March of that year) by local authorities who were persistently absent (those absent for 10% of schooling sessions) during the academic year, in England for financial years 2016/17 - 2021/22.",
+        "publisher": "https://staging.idpd.uk/publishers/department-for-education",
+        "creator": "https://staging.idpd.uk/publishers/department-for-education",
+        "contact_point": {
+          "name": "ofsted",
+          "email": "mailto:informationrequest@ofsted.gov.uk"
+        },
+        "topics": [
+          "https://staging.idpd.uk/topics/children-education-and-skills"],
+        "frequency": "annual",
+        "keywords": [
+          "state-funded",
+          "children-out-of-school",
+          "schools",
+          "local-authority",
+          "subnational",
+          "county",
+          "region"
+        ],
+        "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+        "issued": "2023-05-16T13:35:25Z",
+        "modified": "2023-09-01T12:30:00Z",
+        "spatial_resolution":["E06", "E07", "E08", "E09", "E11"],
+        "spatial_coverage": "E92000001",
+        "temporal_resolution": ["P1Y"],
+        "temporal_coverage": {
+          "start": "2022-01-01T12:00:00Z",
+          "end": "2022-12-31T12:00:00Z"
+        },
+        "versions_url": "https://staging.idpd.uk/datasets/persistent-absences-for-pupils/editions/2023-05/versions",
+        "versions": [
+          {
+            "@id": "https://staging.idpd.uk/datasets/persistent-absences-for-pupils/editions/2023-05/versions/1",
+            "issued": "2023-05-16T13:35:25Z"
+          }
+        ],
+        "next_release": "2024-05-01T09:30:00Z",
+        "table_schema": {
+          "columns": [
+            {
+              "name": "period",
+              "datatype": "datetime",
+              "titles": "Gregorian Time Interval",
+              "description": "The time period associated with the observation."
+            },
+            {
+              "name": "areacd",
+              "datatype": "string",
+              "titles": "ONS Geography Code",
+              "description": "The statistical geography of the observation being measured."
+            },
+            {
+              "name": "observation",
+              "datatype": "decimal",
+              "titles": "Percentage of pupils who have been looked after by the local authorities over financial years 2016/17 - 2021/22",
```

Comment:
Please check this title from the published dataset on https://beta.gss-data.org.uk/datasets, to confirm that it is this long.